### PR TITLE
T40078 Added Jupyterhub documentation

### PR DESCRIPTION
**Pull request by vikram-bitnami**

### CODE REVIEW

File: charts/jupyterhub/_install_modules.md.erb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+title: Add support for other Python libraries
+category: configuration
+weight: 40
+highlight: 40
+=end %>
+
+Follow the steps below to add support for other Python libraries in the JupyterHub deployment:
+
+* Browse to the service IP address and log in using the credentials set or generated at deployment time.
+* Click the JupyterHub logo to return to the list of available notebooks.
```
Comment (rafariossaa) [inline charts/jupyterhub/_install_modules.md.erb:15]: If this the starting point is the user login in, this step is not needed, as the user is redirected to the notebook list when logged in .

File: charts/jupyterhub/_configure_single_user_instances.md.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+apps: jupyterhub
+platforms: kubernetes
+id: configure_single_user_instances
+title: Configure single user instances
+category: configuration
+weight: 40
+highlight: 40
+=end %>
+
+As explained in the [documentation](/xyz/get-started/understand-default-configuration/), the Hub is responsible for deploying the Single User instances. The configuration of these instances is passed to the Hub instance via the *hub.configuration* chart parameter.
```
Comment (rafariossaa) [inline charts/jupyterhub/_configure_single_user_instances.md.erb:12]: Not very sure if this url (`/xyz/....`) is a placeholder.

**Review by rafariossaa — CHANGES_REQUESTED** [review]
Please, check my comments.

File: charts/jupyterhub/_configure_single_user_instances.md.erb (hunk 2 above)
Comment (vikram-bitnami) [reply charts/jupyterhub/_configure_single_user_instances.md.erb]: Thanks, I missed that

**Review by rafariossaa — APPROVED** [review]
LGTM

**Pull request merged**